test/erc20.spec.ts

```diff
@@ -1,128 +1,191 @@
-import { expect } from './setup'
-
+/* External Imports */
 import { ethers } from 'hardhat'
-import { Contract, Signer } from 'ethers'
+import { Signer, ContractFactory, BigNumber } from 'ethers'
+import { JsonRpcProvider } from '@ethersproject/providers'
+
+/* Internal Imports */
+import { expect } from './setup'
+import deploymentsInfo from '../deployments/l1/ERC20.json'
+import { ERC20 } from '../typechain/ERC20'
 
 describe('ERC20', () => {
-  let account1: Signer
-  let account2: Signer
-  let account3: Signer
-  before(async () => {
+  let ERC20: ERC20,
+    provider: JsonRpcProvider,
+    account1: Signer,
+    account2: Signer,
+    account3: Signer
+
+  const name: string = 'Some Really Cool Token Name'
+  const initialSupply: number = 10_000_000
+  const useL2: boolean = (process.env.TARGET === 'ovm')
+
+  // // For the Optimistic test file
+  // const privateKey1: string = ethers.Wallet.createRandom().privateKey
+  // const privateKey2: string = ethers.Wallet.createRandom().privateKey
+  // const privateKey3: string = ethers.Wallet.createRandom().privateKey
```

Comment:
What's the purpose of commenting this snippet out? We may want to either remove it entirely or note why it's useful to keep in the source code.

Reply:
I can move this code block to the README instead. 
My thought was that it's self-explanatory for how it's used with the code comments, and can be easily uncommented out when working with the optimistic test file, and then after, removed from the EVM test file